easy-deploy-script/easy-deploy-viral-ngs.sh

```diff
@@ -197,39 +198,114 @@ function install_miniconda(){
     fi
 }
 
+function install_viral_ngs_conda(){
+    # provide an avenue to specify a package path, or to use a previously-built local package
+    if [ $# -eq 2 ]; then
+        if [ "$2" == "--use-local" ]; then
+            conda install -q -c broad-viral -c r -c bioconda -c conda-forge -c defaults --override-channels -y -p "$VIRAL_CONDA_ENV_PATH" --use-local viral-ngs || exit 1
+            echo "using local...."
+            exit 0
+        else
+            conda install -q -c broad-viral -c r -c bioconda -c conda-forge -c defaults --override-channels -y -p "$VIRAL_CONDA_ENV_PATH" $2 || exit 1
+        fi
+
+    elif [ $# -eq 3 ]; then
+        if [ "$2" == "--viral-ngs-version" ]; then
+            conda install -q -c broad-viral -c r -c bioconda -c conda-forge -c defaults --override-channels -y -p "$VIRAL_CONDA_ENV_PATH" viral-ngs=$3 || exit 1
+        else
+            echo "--viral-ngs-version specified but no version given"
+        fi
+    elif [ $# -eq 1 ]; then
+        conda install -q -c broad-viral -c r -c bioconda -c conda-forge -c defaults --override-channels -y -p "$VIRAL_CONDA_ENV_PATH" viral-ngs || exit 1
+    fi
+}
+
+function install_viral_ngs_git(){
+    if [ ! -L "$VIRAL_NGS_PATH" ]; then
+        git clone https://github.com/broadinstitute/viral-ngs.git "$VIRAL_NGS_PATH"
+    else
+        echo "$VIRAL_NGS_PATH/ symlink already exists. Skipping link."
+    fi
+}
+
+function install_viral_ngs_conda_deps() {
+    conda install -c r -c bioconda -c conda-forge -c defaults --override-channels -y -p $VIRAL_CONDA_ENV_PATH --file "$VIRAL_NGS_PATH/requirements-conda.txt"
```

Comment:
maybe add `-q` to these conda install commands to reduce output verbosity, and a check to exit the script if `conda install` fails